sim/make/scala-build.mk

```diff
@@ -138,7 +138,7 @@ test: $(FIRESIM_MAIN_CP) $(FIRESIM_TEST_CP) $(TARGET_CP)
 
 .PHONY: testOnly
 testOnly: $(FIRESIM_MAIN_CP) $(FIRESIM_TEST_CP) $(TARGET_CP)
-	cd $(base_dir) && java -cp $$(cat $(FIRESIM_TEST_CP)) org.scalatest.run $(SCALA_TEST)
+	cd $(base_dir) && $(SBT_NON_THIN) ";project $(firesim_sbt_project); testOnly $(SCALA_TEST)"
```

Comment:
That doesn't work when running the tests with chipyard-as-top. 
I'm happy to remove those tests from chipyard, howeer.

Reply:
@nandor Any thoughts on this?

Reply:
Those tests should be removed from chipyard. It is important here to not use the JAR as it very significantly slows down tests.

Reply:
Any opposition to adding an alternate testOnly rule that uses the old sbt invocation? I'd prefer to keep the tests in Chipyard, if we can, since Chipyard or rocket chip changes can break things here.

Reply:
That could work. Can't chipyard use FireSim's copy of the tests? They are now duplicated across the two repositories.

Reply:
Yes I think that should work. I'll try this out in this PR. Then we can remove them from Chipyard

Reply:
Could you let me know what command you are running with chipyard-as-top that is failing? Would it be possible to fix it?

Reply:
If we can get chipyard to use the files in FireSim, `https://github.com/ucb-bar/chipyard/pull/1317` could be merged and picked up. Otherwise, I'd be happy with a `scalaTestRun` `.PHONY` rule that uses the cached classpath, reverting the existing `testOnly` rule to use `sbt`.

Reply:
Nvm, I'm not sure why it wasn't working before, it seems fine now. I reverted this.